config/crd/bases/sagemaker.services.k8s.aws_notebookinstances.yaml

```diff
@@ -224,6 +224,10 @@ spec:
               notebookInstanceStatus:
                 description: The status of the notebook instance.
                 type: string
+              stoppedByControllerMETA:
+                description: The URL that you use to connect to the Jupyter notebook
```

Comment:
description is incorrect

Reply:
It's because the file is auto-generated, another reason to use annotations.